cli/src/main/kotlin/StartCommand.kt

```diff
@@ -59,12 +59,14 @@ class StartCommand(private val config: OrtServerOptions) : SuspendingCliktComman
         option(
             "--parameters-file",
             envvar = "ORT_RUNS_START_PARAMETERS_FILE",
-            help = "The path to the Create ORT run configuration file!"
+            help = "The path to a JSON file containing the ORT run configuration " +
```

Comment:
Now with the previous commit dropping "ORT" from "ORT run", consider doing the same here, too.